<launch>
    <node pkg="marker_detector" type="uav_tf_publisher" name="marker_detector" output="screen">
        <param name= "cam_cal_file" type="string" value="$(find marker_detector)/config/camera.xml"/>
        <param name= "init_yaw" type="double" value="0.0"/>
        <remap from="~/image" to="/zed/left/image_rect_color"/>
        <remap from="~/uav_odom" to="/vins_estimator_be_1b/odometry"/>
    </node>
</launch>
